src/sentry/api/event_search.py

```diff
@@ -607,7 +608,7 @@ def convert_search_filter_to_snuba_query(search_filter):
             return condition
 
 
-def get_snuba_query_args(query=None, params=None):
+def get_snuba_filter(query=None, params=None):
```

Comment:
:clap: This is a much better name.

Reply:
I would remove Snuba from the name. The eventstore interface is supposed to become agnostic.